keyboards/crkbd/lib/layer_state_reader.c

```diff
@@ -4,10 +4,10 @@
 #include "crkbd.h"
 
 #define L_BASE 0
-#define L_LOWER 8
-#define L_RAISE 16
-#define L_ADJUST 65536
-#define L_ADJUST_TRI 65560
+#define L_LOWER 2
```

Comment:
Specifically, these changes necessitate the changing of the other keymaps, as this module is hard coded to expect the specific layer values. 

There are other ways to fix this, I'm sure, but this was the simplest option.

Reply:
I'll implement, that actually, since it's .... better in general. Thanks!